integration-tests/ledger-signer-failing-tests.spec.ts

```diff
@@ -55,9 +56,11 @@ CONFIGS().forEach(({ lib, setup }) => {
                 done();
             });
 
-            describe('Should be abble to used Ledger with wallet API', () => {
+            describe('Test that Ledger can be used with wallet API', () => {
+                
                 jest.setTimeout(60000)
-                it('Should throw error when user declines transaction with Ledger', async (done) => {
+
+                it('Verify that Ledger will throw an error when contract origination is declined', async (done) => {
```

Comment:
The operation is a transaction, not orgination

Reply:
changed to
Verify that Ledger will throw an error when a transaction is declined